code/datums/components/tackle.dm

```diff
@@ -192,9 +192,9 @@
 			user.SetKnockdown(0)
 			user.set_resting(FALSE, TRUE, FALSE)
 			user.forceMove(get_turf(target))
-			target.adjustStaminaLoss(40)
-			target.Paralyze(5)
-			target.Knockdown(30)
+			target.adjustStaminaLoss(50)
+			target.Paralyze(8) //Otherwise the victim can just instantly get out of the grab.
```

Comment:
you are aware that this is MEANT to be a passive grab for a reason and not just being an instant upgrade to aggro, yes?

Reply:
Yeah but like, you can essentially escape it from it instantly invalidating the grab completely